src/dataDisplay/Icon/images/bookmark.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default {
+  sm: (
+    <svg
+      height="16"
+      width="12"
+      viewBox="0 0 12 12"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg">
+      <g fill="none" fillRule="evenodd">
+        <path
+          className="icon-stroke"
+          d="M9 11L5 8.22222L1 11V2.11111C1 1.81643 1.12041 1.53381 1.33474 1.32544C1.54906 1.11706 1.83975 1 2.14286 1H7.85714C8.16025 1 8.45094 1.11706 8.66527 1.32544C8.87959 1.53381 9 1.81643 9 2.11111V11Z"
+          strokeWidth="1"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+      </g>
+    </svg>
+  ),
+  md: (
+    <svg
+      height="24"
+      width="20"
+      viewBox="0 0 12 12"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg">
+      <g fill="none" fillRule="evenodd">
+        <path
+          className="icon-stroke"
```

Comment:
We need to add SVGO into the build process. This would be cleaned up.

Reply:
pff yes, apart from this optimization I think we should create a task to rewrite the component because there are svg code duplication everywhere for exposing only 2 sizes (sm, md) what doesn't make sense as this is a vector. Even worse, when you need other size, what is typical, I end up writing an styled component for adjustments

@dasanra